care/facility/tests/test_patient_consultation_api.py

```diff
@@ -0,0 +1,92 @@
+import datetime
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Similar lines in 2 files](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5324897)